### PR TITLE
Add /rise25 page for Mozilla 25th Anniversary 

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary

This PR adds the /rise25 page for mozillas 25th anniversary.

Most of the complexity came from the video at the top of the page, since they wanted a video to be played inline. The video is in webm and mp4 format.

[Design](https://docs.google.com/document/d/1JmfoMKyBBv6UMHCIlVcjn7VOSzWtUbqEwV7cKH59JYQ/edit)
[Copy](https://www.figma.com/file/xmV6xLuY58kUberKsyIBCQ/Rise-25?node-id=199%3A1549&t=aA9OC2zpXydco4zp-1)
[QA Document](https://docs.google.com/spreadsheets/d/1EJTueew7RNUVhbfPjZCaBIL2GzIczN5I3A1rb1Q2edM/edit?usp=sharing)

This should be merged by EOD friday to be launched on Monday morning

## Testing
Demo server URL: https://www-demo4.allizom.org/en-US/rise25

I have not had time to add specific tests for this page, but should be something added as a fast followup

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/rise25/includes/rise25-video.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{#
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/rise25/includes/rise25-video.html:1]: Question: Is this file used anywhere? If not, can it be removed?

File: bedrock/mozorg/templates/mozorg/rise25/includes/rise25-hero.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{#
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/rise25/includes/rise25-hero.html:1]: Question: Is this file being used anywhere? If not, can it be removed?

File: bedrock/mozorg/templates/mozorg/rise25/rise25.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            <img src="/media/img/mozorg/rise25/RISE25.svg" width="171" height="35">
+            <h1>In search of<br/> the visionaries<br/> and builders of<br/> tomorrow</h1>
+            <div class="play-container">
+              <span class="play-button">
+                <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M23.3047 15.0008L10.8493 22.1919L10.8493 7.80973L23.3047 15.0008Z" fill="white" />
+                  <circle cx="15" cy="15" r="14.5908" stroke="white" stroke-width="0.818462" />
+                </svg>
+              </span>
+              <p>Play video</p>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/rise25/rise25.html:79]: Issue (potentially blocking): This `<div>` isn't accessible (via keyboard or screen reader). Might it be possible to use a `<button>` element instead maybe?

File: media/css/mozorg/rise25.scss [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+        grid-template-columns: 1fr 1fr;
+    }
+
+    @media #{$mq-lg} {
+        grid-template-columns: repeat(3, 1fr);
+    }
+}
+
+.nomination-card {
+    background-color: var(--card-color, #{$color-black});
```
Comment (alexgibson) [inline media/css/mozorg/rise25.scss:278]: Suggestion (non-blocking): Is it worth adding a fallback `background-color` above this line for older browsers? They could probably all have the same color to make things simple? Without it there's some contrast issues:

image

File: media/img/mozorg/rise25/r25_hero_1312-2x.jpg
Comment (alexgibson) [inline media/img/mozorg/rise25/r25_hero_1312-2x.jpg:1]: Suggestion (non-blocking): looks like I can shave around 20-30% off the file sizes for many of the images here using tinypng?

The hero image seems to be the most costly ones, with some variants come in at around 500kb.

File: bedrock/mozorg/templates/mozorg/rise25/rise25.html (hunk 3 above)
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/rise25/rise25.html:79]: Question (non-blocking): do we want any GA events around the video play interaction?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Looks great, nice work!

There are a couple of accessibility issues with the video play button and social sharing links that would be nice to get fixed, but even those could be a fast follow next week if we need to get things out the door?

Other than that, mostly just a few nits and things to tidy up / optimize.

File: bedrock/mozorg/templates/mozorg/rise25/includes/rise25-social-share.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    <li>
+      <a class="email">
+        <svg title="Share with email" width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" clip-rule="evenodd"
+            d="M3.1942 2.86523H13.4665C14.1727 2.86523 14.7506 3.44305 14.7506 4.14927V11.8535C14.7506 12.5597 14.1727 13.1376 13.4665 13.1376H3.1942C2.48797 13.1376 1.91016 12.5597 1.91016 11.8535V4.14927C1.91016 3.44305 2.48797 2.86523 3.1942 2.86523Z"
+            stroke="currentColor" stroke-width="1.28115" stroke-linecap="round" stroke-linejoin="round" />
+          <path d="M14.7506 4.14941L8.33036 8.64356L1.91016 4.14941" stroke="currentColor" stroke-width="1.28115"
+            stroke-linecap="round" stroke-linejoin="round" />
+        </svg>
+      </a>
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/rise25/includes/rise25-social-share.html:52]: Question: do we also want any GA events on the social share interactions?

File: media/css/mozorg/rise25.scss (hunk 4 above)
Comment (nathan-barrett) [reply media/css/mozorg/rise25.scss:278]: woof, yeah. Lets fix that

**Pull request merged**